pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -287,12 +280,12 @@ class OkHttpClientWithProfile extends OkHttpClient {
   OkHttpClientWithProfile() : super();
 }
 
-extension on Uint8List {
+extension JUint8List on Uint8List {
   JArray<jbyte> toJArray() =>
       JArray(jbyte.type, length)..setRange(0, length, this);
 }
 
-extension on JArray<jbyte> {
+extension DartJArray on JArray<jbyte> {
   Uint8List toUint8List({int? length}) {
     length ??= this.length;
     final list = Uint8List(length);
```

Comment:
This can be done without a for loop. `JArray<jbyte>` also has a `getRange` method that you can use and it will be faster.

Reply:
Done! Thanks!